src/components/registration-view/registration-view.jsx

```diff
@@ -13,9 +15,23 @@ export function RegistrationView(props) {
   const handleSubmit = (e) => {
     e.preventDefault();
     console.log(username, email, birthday, movies, password);
-    props.endRegister();
   };
 
+  axios.post('https://movie-api-1684.herokuapp.com/users', {
+    Username: username,
+    Password: password,
+    Email: email,
+    Birthday: birthday
+  })
+  .then(response => {
+    const data = response.data;
+    console.log(data);
+    window.open('/', '_self'); // the second argument '_self' is necessary so that the page will open in the current tab
+  })
+  .catch(e => {
+    console.log('error registering the user')
+  });
+
```

Comment:
Move the whole `axios.post` code block to the `handleSubmit` function. Because you want to POST your data when the `register` button is clicked

Reply:
const handleSubmit = (e) => {
    e.preventDefault();
  
    axios.post('https://movie-api-1684.herokuapp.com/users', {
      Username: username,
      Password: password,
      Email: email,
      Birthday: birthday
    })
  .then(response => {
     const data = response.data;
     console.log(data);
     window.open('/', '_self'); 
  })
  .catch(e => {
    console.log('error registering the user')
   })
 };

ok now is like this